segmentation/utils/visualize.py

```diff
@@ -18,3 +18,84 @@ def visualize_z_scores(z, pheno_titles):
     zplot = pd.DataFrame(z, columns=pheno_titles, index=pheno_titles)
     sns.set(font_scale=.7)
     sns.clustermap(zplot, figsize=(8, 8), cmap="vlag")
+    
+def newDict(names):
+    result = {}
+    for name in names:
+        result[name] = [0]
+    return result
+
+def visualize_distribution_of_cell_count(df, id_col_name, cell_col_name):
```

Comment:
Please add multiline comment/description of function.  It should contain a blurb about what the function does, its inputs + their format, and its outputs (returns/write out to disk) + their formats.  Examples can be found in most of the other scripts in the `utils` folder.

Also, you should change `df` to a more descriptive name for better readability.